js/filters/roseGold.js

```diff
@@ -0,0 +1,15 @@
+/* global app */
+window.filters.roseGold = {
+  name: 'rose gold',
+  menu: 'Edit',
+  run: function () {
+    const imageData = app.ctx.getImageData(0, 0, app.canvas.width, app.canvas.height)
+    const data = imageData.data
+    for (let i = 0; i < data.length; i += 4) {
+      data[i] = 255 * 0.718
+      data[i + 1] = 255 * 0.431
+      data[i + 2] = 255 * 0.475
```

Comment:
`255 * 0.718` equals `183.09`, every time, so u don't need to do arithmetic here, u might as well just write `data[i] = 183.09` (same for the green && blue channel), but the bigger issue here is that the color it generates (that rose gold color) is applied wholesale to every pixel that isn't transparent. so this filter just turns the entire image a solid color.

if u meant to give the entire canvas a rose gold "tint" rather than turning it into a solid color, u'll need to take a different approach, consider @shmoopyyy code for her [pink filter](https://github.com/shmoopyyy/paintArtware2.1/blob/main/js/filters/pink.js)